h/services/search_index/_queue.py

```diff
@@ -1,27 +1,25 @@
 from collections import Counter
 from datetime import datetime
 
-from celery.utils.log import get_task_logger
 from dateutil.parser import isoparse
 from sqlalchemy import func, select, text
 from zope.sqlalchemy import mark_changed
 
 from h.db.types import URLSafeUUID
 from h.models import Annotation, Job
 
-logger = get_task_logger(__name__)
-
-# Strings used in log messages.
-DELETED_FROM_DB = "Jobs deleted because annotations were deleted from the DB"
-MISSING = "Annotations synced because they were not in Elasticsearch"
-OUT_OF_DATE = "Annotations synced because they were outdated in Elasticsearch"
-UP_TO_DATE = "Jobs deleted because annotations were up to date in Elasticsearch"
-FORCED = "Annotations synced because their jobs had force=True"
-
 
 class Queue:
     """A job queue for synchronizing annotations from Postgres to Elastic."""
 
+    class Result:
+        # Values for reporting which should stringify nicely
+        DELETED_FROM_DB = "Jobs deleted because annotations were deleted from the DB"
+        MISSING = "Annotations synced because they were not in Elasticsearch"
+        OUT_OF_DATE = "Annotations synced because they were outdated in Elasticsearch"
+        UP_TO_DATE = "Jobs deleted because annotations were up to date in Elasticsearch"
+        FORCED = "Annotations synced because their jobs had force=True"
```

Comment:
Made this enum like, but not actually an enum, as that prints out a little weird. Still legible, but not as nice.

Nesting it in like this is a Java like trick, and means you can access the enum wherever the `Queue` goes. This is convenient for the caller as it's usually them that want to make comparisons with the values (although we don't actually use if for that here)